OrbitCore/Callstack.h

```diff
@@ -7,20 +7,24 @@
 
 #include <xxhash.h>
 
-#include <string>
 #include <vector>
 
 #include "CallstackTypes.h"
 
 struct CallStack {
```

Comment:
Make it `class` as it has private members and add `public:` for the public members.